.travis.yml

```diff
@@ -5,8 +5,8 @@ cache: bundler
 # For example, if our gemspec says `required_ruby_version = ">= 2.3.0"`, and
 # ruby 2.5.0 has just been released, then we test 2.3 and 2.5, but not 2.4.
 rvm:
-  - 2.3.8
-  - 2.6.0
+  - 2.3
+  - 2.7
```

Comment:
This should make maintenance of the Ruby versions a bit easier. As Travis adds newer Ruby versions, this will pick the newest versions available there. WDYT?

Reply:
Sounds good to me. Patch levels of ruby seem unlikely to break PT.